fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/OnboardingPageState.kt

```diff
@@ -9,16 +9,16 @@ import androidx.annotation.DrawableRes
 /**
  * Model containing data for [OnboardingPage].
  *
- * @param image [DrawableRes] displayed on the page.
- * @param title [String] title of the page.
- * @param description [String] description of the page.
- * @param linkTextState [LinkTextState] part of description text with a link.
- * @param primaryButton [Action] action for the primary button.
- * @param secondaryButton [Action] action for the secondary button.
- * @param onRecordImpressionEvent Callback for recording impression event.
+ * @property imageRes [DrawableRes] displayed on the page.
+ * @property title [String] title of the page.
+ * @property description [String] description of the page.
+ * @property linkTextState [LinkTextState] part of description text with a link.
+ * @property primaryButton [Action] action for the primary button.
+ * @property secondaryButton [Action] action for the secondary button.
+ * @property onRecordImpressionEvent Callback for recording impression event.
  */
 data class OnboardingPageState(
-    @DrawableRes val image: Int,
+    @DrawableRes val imageRes: Int,
```

Comment:
This change is creating a lot of changes in lots of files that are not required for this patch. Could we not do this on this PR?